docs/user_load_testing.md

```diff
@@ -0,0 +1,84 @@
+# User load testing
+
+In order to prepare for Closed Beta and General Availability, the Matrix Service is in need of a solution to profile its
+performance in terms of real world usage. A user load testing framework that can measure performance across a number of
+the service's control inputs can achieve this. The following provides concrete definitions for the terms _performance_ 
+and _control inputs_:
+
+### Performance
+
+Performance, in context of the User Load Tests, refers to the processing time of a matrix request measured from the
```

Comment:
Refer to caching here and relevance to the concurrent user test?

Reply:
Sure thing (currently being handled by selecting a random subset of matrixable bundles of the requested input size to lower the chance of a cache hit)